CODEOWNERS

```diff
@@ -29,6 +29,7 @@
 /code/addons/storyshots-core/      @ndelangen
 /code/addons/storyshots-puppeteer/ @ndelangen
 /code/addons/storysource/          @ndelangen
+/code/addons/themes/               @JReinhold @Integrayshaun
```

Comment:
Are you sure that you want to also be on the hook for this addon? ;)

Reply:
tag team is best team!